### DISCUSSION TITLE
What to put in Supabase site URL box?

### DISCUSSION THREAD

**Discussion by SeriousLee20** [vercel/community#5238] [Help]
### Summary

I'm developing a web app using Nuxt3 and Supabase. The project is using OAuth (Google/GitHub).
The project is running normally on the local host.

However, when I deploy it to Vercel, it becomes a problem to redirect back from the OAuth link when I set the site URL to any link other than http://localhost:3000/.
I tried to change the site URL with

https://[project-name]-[id generated by vercel in login page]-[username].vercel.app/
https://[project-name].vercel.app/
I also tried the URLs above with redirectTo option in supabase.auth.login function.

In the end, I still can't find out what should I put in the site URL box. Is there anything I'm missing out or understanding wrongly?

### Steps to Reproduce

URLs I tried with site URL:
<img width="388" alt="image" src="https://github.com/vercel/community/assets/71384383/03c98095-451d-4d03-a3f8-b2b78d839e56">

**The redirectTo option in Supabase auth**
`const login = async () => {
  const { data, error } = await auth.signInWithOAuth({
    provider: "google",
    options: {
      redirectTo: "https://pro-con.vercel.app/overview",
    },
  });
`
Error 405 returned by using above approach.

**Comment by jonaaldas** [comment]
Did you find a solution?
